tsdb/engine/tsm1/engine.go

```diff
@@ -1843,7 +1843,7 @@ func (e *Engine) createVarRefSeriesIterator(ref *influxql.VarRef, name string, s
 		for i, ref := range opt.Aux {
 			// Create cursor from field if a tag wasn't requested.
 			if ref.Type != influxql.Tag {
-				cur := e.buildCursor(name, seriesKey, &ref, opt)
+				cur := e.buildCursor(name, seriesKey, tfs, &ref, opt)
```

Comment:
Instead of passing this in, the series key should probably just get reparsed inside of `buildCursor`. Otherwise, I feel like the arguments are redundant. Not necessary though. If this is a hotspot, one fewer allocation is likely better anyway.